oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/property/PropertyIndexEditor.java

```diff
@@ -328,6 +332,7 @@ private void checkUniquenessConstraints() throws CommitFailedException {
                     String msg = String.format(
                             "Uniqueness constraint violated property %s having value %s",
                             propertyNames, failed);
+                    log.warn(msg);
```

Comment:
I'm wondering if we should add a prefix to the log. Without the prefix it might be hard for down-stream logging to distinguish between the two sources of this message (the log.warn and the exception), as they contain a large common part. What about something like below (or maybe a better variant of it) ?
```suggestion
                    log.warn("checkUniquenessConstraints: " + msg);
```

Reply:
Done